### DISCUSSION TITLE
RIA in footer?

### DISCUSSION THREAD

**Discussion by brianzelip** [Relay-Maryland/relaymaryland.com#16] [General]
@j-p-courneya  Should the footer highlight "Relay Improvement Association" as the source behind the website? Or should it just read "Relay, Maryland"?

<img width="813" alt="relay footer" src="https://github.com/Relay-Maryland/relaymaryland.com/assets/3411019/f5b4140a-410f-43eb-bef5-6668641b6e22">

**Comment by brianzelip** [comment]
Also fyi - that github logo was meant to be temporary, as a useful link for me to get to the repo quickly during initial development.

**Reply by j-p-courneya** [reply]
Aight cool, swap out with insta at some point then. Although the link to GitHub is fashionable

**Reply by brianzelip** [reply]
@j-p-courneya What is the instagram handle?

**Reply by j-p-courneya** [reply]
@brianzelip 

https://www.instagram.com/relaymaryland?igsh=OW5zN3A4NHR6OTV6

**Comment by j-p-courneya** [comment]
I think that having it there is good and allows for ria to sponsor it.
